pkgs/os-specific/bsd/setup-hook.sh

```diff
@@ -49,7 +49,6 @@ addMakeFlags() {
   makeFlags="LIBDIR=${!outputLib}/lib $makeFlags"
   makeFlags="SHLIBDIR=${!outputLib}/lib $makeFlags"
   makeFlags="SHAREDIR=${!outputLib}/share $makeFlags"
-  makeFlags="MANDIR=${!outputMan}/share/man $makeFlags"
```

Comment:
hmmm - this seems to be an @artemist commit, https://github.com/rhelmot/nixpkgs/commit/3e2e56f462fc048e63f5d201f0b882536618adba. Files from it might be missing from this PR...

Reply:
FreeBSD treats it a bit differently. On NetBSD you might set `MANDIR=/usr/share/man` then put man files in `$(MANDIR)/man3`, on FreeBSD you'd set `MANDIR=/usr/share/man/man` then put man files in `$(MANDIR)3`
I added the original back to NetBSD but it may not have made it into this PR.

Reply:
OK makes sense